README.md

```diff
@@ -14,4 +14,4 @@ The official repository for the YARG launcher (a.k.a., YAL or Yet Another Launch
 4. Type in `git clone https://github.com/YARC-Official/YARG-Launcher.git`.
```

Comment:
Discrepancy detected (I know you didn't do this, but should be `git clone https://github.com/YARC-Official/YARC-Launcher.git`